docs/001_develop/02_server-capabilities/018_model_context_protocol/index.mdx

```diff
@@ -0,0 +1,201 @@
+---
+title: 'Model Context Protocol - Overview'
+sidebar_label: 'MCP - Model Context Protocol'
+id: model-context-protocol
+keywords: [AI, MCP, model, context, protocol]
+tags:
+- AI
+- MCP
+- model
+- context
+- protocol
+sidebar_position: 18
+---
+
+# MCP - Model Context Protocol
+
+Genesis now supports exposing application endpoints through **MCP (Model Context Protocol)**, enabling seamless interaction between your applications and Large Language Models (LLMs). This new capability allows any LLM (e.g., ChatGPT, Claude, Gemini) to understand and invoke functionality within your Genesis-built apps using structured interfaces.
```

Comment:
This is a real nitpick but the way your phrase this as a new feature means it'll not read quite right in a few months when it's not such a new feature. Not sure if that matters but I thought I would point it out. I do see we talk about it as an experiemntal feature so if the intention is to come to these docs later and clean them up then no issue